hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -1198,11 +1198,23 @@
         { bits: "6:0",
           desc: '''This field will set the threshold that the depth of the Observe FIFO
                 will be compared with when setting the interrupt status bit.
+                Note: a value of zero is reserved and not to be used.
                 '''
           resval: "0x20"
         }
       ]
     },
+    { name: "OBSERVE_FIFO_DEPTH",
+      desc: "Observe FIFO depth register",
+      swaccess: "ro",
```

Comment:
just one note. If the observe depth is already coming out of a set of flops, you could declare this as `hwext`, and basically allow those flops to directly drive the read back value instead of flopping them again.

Reply:
interesting...I rarely think about saving 6 flops in my high-speed ASIC designs, but this seems easy enough. Will do!